Workbooks/ServiceBus/AtResource/ResourceInsights.workbook

```diff
@@ -0,0 +1,2406 @@
+                      },
+                      {
+                        "namespace": "microsoft.servicebus/namespaces",
+                        "metric": "microsoft.servicebus/namespaces--ConnectionsOpened",
+                        "aggregation": 1,
+                        "columnName": "Connections Opened (Sum)"
+                      },
+                      {
+                        "namespace": "microsoft.servicebus/namespaces",
+                        "metric": "microsoft.servicebus/namespaces--SuccessfulRequests",
+                        "aggregation": 1,
+                        "columnName": "Successful Requests (Sum)"
+                      },
+                      {
+                        "namespace": "microsoft.servicebus/namespaces",
+                        "metric": "microsoft.servicebus/namespaces--ThrottledRequests",
+                        "aggregation": 1,
+                        "columnName": "Throttled Requests (Sum)"
+                      },
+                      {
+                        "namespace": "microsoft.servicebus/namespaces",
+                        "metric": "microsoft.servicebus/namespaces--IncomingMessages",
+                        "aggregation": 1,
+                        "columnName": "Incoming Messages (Sum)"
+                      }
+                    ],
+                    "gridFormatType": 1,
+                    "tileSettings": {
+                      "titleContent": {
+                        "columnMatch": "Metric",
+                        "formatter": 1,
+                        "formatOptions": {}
+                      },
+                      "leftContent": {
+                        "columnMatch": "Value",
+                        "formatter": 12,
+                        "formatOptions": {
+                          "palette": "auto"
+                        },
+                        "numberFormat": {
+                          "unit": 17,
+                          "options": {
+                            "maximumSignificantDigits": 3,
+                            "maximumFractionDigits": 2
+                          }
+                        }
+                      },
+                      "showBorder": false
+                    },
+                    "gridSettings": {
+                      "rowLimit": 10000
+                    }
+                  },
+                  "name": "Summary Metrics"
+                },
+                {
+                  "type": 10,
+                  "content": {
+                    "chartId": "workbookbed4014b-7f7f-46fa-a61f-77e3221dd950",
+                    "version": "MetricsItem/2.0",
+                    "size": 0,
+                    "chartType": 2,
+                    "resourceType": "microsoft.servicebus/namespaces",
+                    "metricScope": 0,
+                    "resourceParameter": "Resource",
+                    "resourceIds": [
+                      "{Resources}"
```

Comment:
nit: I see some of these still have {Resources} can you change it to just {Resource}